web/flat-web/deploy/nginx/nginx.conf

```
@@ -15,6 +15,19 @@ http {
     listen 80;
     root /app;
 
+    gzip on;
+    gzip_disable "MSIE [1-6]\.";
+    gzip_min_length 64;
+    gzip_buffers 16 8k;
+    gzip_comp_level 6;
+    gzip_static on;
+    gzip_http_version 1.1;
+    gzip_vary on;
+    gzip_proxied expired no-cache no-store private auth;
+    gzip_types application/javascript application/x-javascript application/json application/manifest+json application/xhtml+xml application/xml video/mp4 font/eot font/otf font/ttf image/svg+xml image/jpeg image/gif image/png text/css text/javascript text/plain text/xml;
```

```suggestion
    gzip_types application/javascript application/x-javascript text/javascript;
```

We only need `js` (Current stage)